packages/web-components/src/slider/slider.styles.ts

```diff
@@ -22,127 +21,98 @@ import {
  * @public
  */
 export const styles = css`
-  ${display('inline-grid')} :host {
-    --thumb-size: 18px;
-    --thumb-padding: 3px;
-    --thumb-translate: calc(var(--thumb-size) * -0.5 + var(--track-width) / 2);
-    --track-overhang: -2px;
-    --track-width: 4px;
-    --fast-slider-height: calc(var(--thumb-size) * 10);
+  ${display('inline-grid')}
+
+  :host {
+    --thumb-size: 20px;
+    --track-margin-inline: calc(var(--thumb-size) / 2);
+    --track-size: 4px;
+    --track-overhang: calc(var(--track-size) / -2);
     --slider-direction: 90deg;
+    --border-radius: ${borderRadiusMedium};
+    --step-marker-inset: var(--track-overhang) -1px;
+    position: relative;
     align-items: center;
+    justify-content: center;
     box-sizing: border-box;
     outline: none;
-    cursor: pointer;
     user-select: none;
-    border-radius: ${borderRadiusSmall};
-    touch-action: pan-y;
-    min-width: calc(var(--thumb-size) * 1px);
-    width: 100%;
+    touch-action: none;
+    min-width: 120px;
+    min-height: 32px;
+    grid-template-rows: 1fr var(--thumb-size) 1fr;
+    grid-template-columns: var(--track-margin-inline) 1fr var(--track-margin-inline);
+  }
+
+  :host(:not(:disabled)) {
+    cursor: pointer;
   }
 
-  :host([size='small']) {
-    --thumb-size: 14px;
-    --track-width: 2px;
-    --thumb-padding: 3px;
+  :host(:dir(rtl)) {
+    --slider-direction: -90deg;
   }
 
-  :host([orientation='vertical']) {
-    --slider-direction: 0deg;
-    height: 160px;
-    min-height: var(--thumb-size);
-    touch-action: pan-x;
-    padding: 8px 0;
-    width: auto;
-    min-width: auto;
+  :host(${smallState}) {
+    --thumb-size: 16px;
+    --track-overhang: -1px;
+    --track-size: 2px;
+    --border-radius: ${borderRadiusSmall};
   }
 
-  :host([disabled]:hover) {
-    cursor: initial;
+  :host(${verticalState}) {
+    --slider-direction: 0deg;
+    --step-marker-inset: -1px var(--track-overhang);
+    min-height: 120px;
```

Comment:
120px seems to be common for either the height for vertical and the width for horizontal, should we add another CSS property like `--slider-length` at `120px` and refer to that?

Reply:
This is on my TODO list for future improvement, I'll try to convert all styles into logical properties and switching between horizontal and vertical by using `writing-mode`. But wanted to limit the scope for this PR so didn't add that in.